src/libs/Notification/PushNotification/backgroundRefresh/index.android.js

```diff
@@ -0,0 +1,40 @@
+import Onyx from 'react-native-onyx';
+import * as App from '../../../actions/App';
```

Comment:
This causes console error:

<img width="340" alt="Screenshot 2023-08-13 at 10 26 16 PM" src="https://github.com/Expensify/App/assets/108292595/fe806798-9e5c-4f4f-a1f3-37301e7149ce">